### ISSUE TITLE
doesn't compile after recent rocksdb update

### ISSUE THREAD

**Issue by ingvarbiz** [twmht/python-rocksdb#38]
[Remove PlainTable's feature store_index_in_file (#4914)](https://github.com/facebook/rocksdb/commit/ee1818081ff4ca2a49a48cb4ca5b97665b8dcddf) introduced yesterday breaks compilation of python-rocksdb:

```rocksdb/_rocksdb.cpp: In function 'int __pyx_pf_7rocksdb_8_rocksdb_17PlainTableFactory___init__(__pyx_obj_7rocksdb_8_rocksdb_PlainTableFactory*, PyObject*, PyObject*, PyObject*, PyObject*, PyObject*, PyObject*, PyLongObject*, PyLongObject*)':
rocksdb/_rocksdb.cpp:13542:25: error: 'struct rocksdb::PlainTableOptions' has no member named 'store_index_in_file'
__pyx_v_table_options.store_index_in_file = __pyx_t_8;
```

**Comment by ingvarbiz** [comment]
I built python-rocksdb with rocksdb 5.17.2 which is the latest stable release now. Seems to work correctly, however I have not done extensive tests.

**Comment by iFA88** [comment]
5.17.2 has the Remove PlainTable's feature store_index_in_file (#4914) commit, so it is works now or not?

**Comment by ingvarbiz** [comment]
> 5.17.2 has the `Remove PlainTable's feature store_index_in_file (#4914)` commit, so it is works now or not?

5.17.2 released on 2018/11/12 does NOT have the `Remove PlainTable's feature store_index_in_file (#4914)` commit introduced on 2019/01/28.

**Comment by twmht** [comment]
fix with version `0.7.0`
